gui/src/launcher.rs

```diff
@@ -177,6 +178,19 @@ async fn check_network_datadir(mut path: PathBuf, network: Network) -> Result<Ne
         )
     })?;
 
+    if let Some(exe_config) = cfg.internal_bitcoind_exe_config {
+        let args = vec![
+            format!("-chain={}", &network.to_core_arg()),
+            format!("-datadir={}", exe_config.data_dir.to_string_lossy()),
+        ];
+        info!("Starting internal bitcoind");
+        let _ = std::process::Command::new(exe_config.exe_path)
+            .args(&args)
+            .stdout(std::process::Stdio::null())
```

Comment:
Note to (self and other) reviewers: it's fine to discard stdout, we still get `bitcoind`'s logs in the `debug.log`.

Reply:
Shall I add this as a comment in the code?

Reply:
Sure, good idea.

Reply:
As part of https://github.com/wizardsardine/liana/pull/592/commits/6072da8837c4d8daac1c612efb41cbe9da671d22, I introduced a `start_internal_bitcoind()` function that includes a comment about `debug.log`.